docs/en/18.Support for Unseal Tasks.md

```diff
@@ -0,0 +1,119 @@
+## Support for Unseal Tasks
+
+### Overview
+
+Order retrieval is an important part of the Filecoin ecosystem loop. Currently, `Droplet` will use the `Piece data` in its `PieceStore` by default to respond to data requests from retrieval requests. Users can also configure Droplet to control the cached Piece data, so when Droplet finds no data for the target Piece in the database, it will trigger an `unseal task` and issue the task to `Damocles`. 
```

Comment:
```suggestion
Data retrieval is an important part of the Filecoin ecosystem loop. Currently, `Droplet` will use the `Piece data` in its `PieceStore` by default to respond to data requests from retrieval requests. Users can also configure `Droplet` to control the cached Piece data, so when `Droplet` finds no data for the target Piece in the database, it will trigger an `unseal task` and issue the task to `Damocles`. 
```